evaluation/hpc/batch_synthesizer.sh

```
@@ -0,0 +1,28 @@
+# Activate your pre-created conda environment (replace "myenv" with your env name)
+conda activate policywonk
+
+# Start ollama in the background
+export OLLAMA_HOST=0.0.0.0 && ~/ollama/bin/ollama serve &
+
+# (Optional) Ensure the required packages are installed.
+# Uncomment the next line if you need to install from requirements.txt:
+# pip install -r ~/policywonk/requirements.txt
+
+# Change directory to your evaluation folder
+cd ~/policywonk/evaluation
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)